src/scripts/main.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
-// write code here
+const List = document.querySelectorAll('li');
+let sortedList;
+
+function sortList(list) {
+  sortedList = [...list].sort((a, b) =>
+    +b.attributes[1].value.replace(/[^\d]/g, '')
+    - +a.attributes[1].value.replace(/[^\d]/g, ''));
+
+  const ul = document.querySelector('ul');
+
+  ul.innerHTML = sortedList.map(li => `
+    <li>
+      ${li.innerText}
+    </li>
+  `).join('');
```

Comment:
Try do not use innerHTML. Use [append](https://developer.mozilla.org/en-US/docs/Web/API/Element/append) instead